tests/helpers.cmake

```diff
@@ -104,76 +104,70 @@ function(execute_cdb SRC_VERSION SCENARIO)
 	endif()
 endfunction()
 
-function(test_intr_tx prepare_files)
-	set(curr_scenario 0)
-	set(last_scenario 9)
-
-	list(LENGTH VERSIONS num)
-	math(EXPR num "${num} - 1")
+function(test_intr_tx prepare_files curr_version next_version)
```

Comment:
You are changing the number of arguments function takes without updating callers.
I see you did this in a separate commit, but this is bad practice because it breaks bisection.

Reply:
IOW, each commit must build and tests must pass.

Reply:
Done.